### DISCUSSION TITLE
NBC Sports Boston Removed from Xfinity Cable Card Service

### DISCUSSION THREAD

**Discussion by trk2** [OpenSageTV/Community#167] [General]
NBC Sports Boston (and presumably the other NBC sports regionals?) is no longer available over cable card as of 10/24.  This is the first time Comcast has removed active channels from my cable card lineup.  Previously only a select few new channels weren't available for cable card.  I was told by Xfinity that cable card is approaching EOL and more channels may also leave soon.  

I only have tv service for sporting events so this change was the end of my relationship with Comcast, which overall had good support for cable card over the years.  I'm trying Fubo and may also try Channels to get Fubo into Sage.  It's frustrating that they did this change right at the start of the NBA season and disrupted a recording service that has been rock solid for years.

**Comment by mickgeller** [comment]
Long time Comcast customer near Phila. here. The Local NBC Sports channel is still available on CableCard here. Did they convert your channel to an IPTV channel or remove it entirely? If they removed it entirely, I am not sure what the next steps could be. 

Check the Xfinity Streaming App. If the channel is there, then you have options. Channels DVR becomes your new best friend as it works best with streaming sourced input. In addition, you can output from Channels into SageTV, if you want to. It takes effort to set it up, but it is all one-time work. The old forum has the details on how to do it on the Hardware Support Subforum. You will also need the updated XMLTV plugin from the SageTV EPG Service subforum. This is because Channels not only provides the streams, it takes care of the guide data also.
